tfac/tests/test_import.py

```diff
@@ -30,12 +30,13 @@ def test_formMissing():
 
     assert isinstance(tensor_slices, list)
 
-    for i in range(3):
-        assert isinstance(tensor_slices[i], np.ndarray)
-        assert np.any(np.isnan(tensor_slices[i]))
+    for dd in tensor_slices:
+        assert isinstance(dd, np.ndarray)
+        assert np.any(np.isnan(dd))
+        assert np.any(np.isfinite(dd)) # At least one value should be finite
```

Comment:
I added this test to demonstrate what I'm seeing. The tensor slices are just filled with nan's. It's possible I messed this up with the adjustments I made this morning, but I'm not able to find where that might be if so. Could you take a look @scottdtaylor95 ?